src/lib/rng/rng.h

```diff
@@ -30,6 +30,22 @@ class Entropy_Sources;
 */
 class BOTAN_PUBLIC_API(2, 0) RandomNumberGenerator {
    public:
+      /**
+      * Userspace RNGs like HMAC_DRBG will reseed after a specified number
+      * of outputs are generated. Set to zero to disable automatic reseeding.
+      */
+      static constexpr size_t DefaultReseedInterval = 1024;
```

Comment:
Should this maybe live in the `Stateful_RNG` subclass? (no strong opinion, having it all in the same place is nice for sure)

Reply:
Possibly the value is relevent for a stateful RNG that is not deriving from `Stateful_RNG`, for example because the state is not in our address space but lives in some external service process.